ptypes/regen.sh

```diff
@@ -1,10 +1,12 @@
-#!/bin/bash -e
+#!/bin/bash
 #
 # This script fetches and rebuilds the "well-known types" protocol buffers.
 # To run this you will need protoc and goprotobuf installed;
 # see https://github.com/golang/protobuf for instructions.
 # You also need Go and Git installed.
 
+set -Ee
```

Comment:
There's O(few) people that will ever run this script. They can just run it directly instead of subverting the shebang line.